src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="largeForm"
+      action="/"
+      method="post"
+      data-qa="big"
+    >
+      <img
+      class="firstImage"
+      src="./images/Search.svg"
+      alt="search"
+      >
+
+      <input
+      type="text"
+      class="largeInput"
+      placeholder="Try &quot;Los Angeles&quot"
+      data-qa="keypress"
+      >
+    </form>
+
+    <form
+      class="smallForm"
+      action="/"
+      method="post"
+      data-qa="small"
+    >
+      <img
+      src="./images/Search.svg"
+      alt="search"
+      class="secondImage"
+      >
+      <input
```

Comment:
```suggestion
      >
      
      <input
```
don't forget to add space between one-level elements